drivers/sdcard_spi/include/sdcard_spi_params.h

```diff
@@ -70,6 +70,39 @@ static const  sdcard_spi_params_t sdcard_spi_params[] = {
     SDCARD_SPI_PARAMS
 };
 
+/**
+ * @name    Set retry parameters for specific actions
+ *
+ * Retry timeouts in microseconds for specific actions.
+ * The value interpretation is uint32_t.
+ * A value of 0 disables retries.
+ *
+ * @{
+ */
+#ifndef INIT_CMD_RETRY_US
+#define INIT_CMD_RETRY_US           (250000) /**< initialization command retry */
```

Comment:
```suggestion
#define INIT_CMD_RETRY_US           (250 * USEC_PER_MSEC) /**< initialization command retry */
```

This will also make the define `unsigned long`, so you can drop the casts later on.

Reply:
I revised this in a fixup.